ui/imports/shared/views/chat/InvitationBubbleView.qml

```diff
@@ -69,158 +70,144 @@ Item {
         }
     }
 
-    Loader {
-        id: loader
+    background: Rectangle {
+        radius: d.radius
+        color: Style.current.background
+        border.color: Style.current.border
+        border.width: 1
+    }
 
-        active: !!d.invitedCommunity
```

Comment:
What happens now when community details are not fetched yet?

Reply:
I just checked it myself, it works fine but I don't understand why. Most likely `InvitationBubbleView` is instantiated only when the community is fetched, which makes sense. If that's the case, then why we would need this: https://github.com/status-im/status-desktop/commit/6b9b5f20a807e38c3b76f3d81807d2874baae4ed#diff-08a51598e69bdd9997718cb97efe31d0fd4bd5f6132ddb88720dd402e7b21ad2?

If in some scenario `d.invitedCommunity` is null then we'll end up in 1) wrongly rendered bubble 2) a lot of warnings

Reply:
Because the whole thing itself is wrapped inside another Loader that gets reloaded when the parent message gets scrolled away (either via LinksMessageView.qml or StatusMessage.qml); so we always have a `communityId` and we fetch the info on creation

Reply:
1) What will happen to `InvitationBubbleView` if `d.invitedCommunity` is null? 
2) Do we still need this https://github.com/status-im/status-desktop/commit/6b9b5f20a807e38c3b76f3d81807d2874baae4ed#diff-08a51598e69bdd9997718cb97efe31d0fd4bd5f6132ddb88720dd402e7b21ad2 then?

Reply:
> 1. What will happen to `InvitationBubbleView` if `d.invitedCommunity` is null?

Right, it should probably be hidden (`height: 0` or even `visible: false`), just like previously with the Loader

> 2. Do we still need this [6b9b5f2#diff-08a51598e69bdd9997718cb97efe31d0fd4bd5f6132ddb88720dd402e7b21ad2](https://github.com/status-im/status-desktop/commit/6b9b5f20a807e38c3b76f3d81807d2874baae4ed#diff-08a51598e69bdd9997718cb97efe31d0fd4bd5f6132ddb88720dd402e7b21ad2) then?

That I don't know; I haven't added or changed that part

Reply:
> Right, it should probably be hidden (`height: 0` or even `visible: false`), just like previously with the Loader

(not critical) Still, wouldn't it result in a lot of warnings? :thinking:

Reply:
Nope, I just tried :) no warnings